vumi/persist/tests/test_model.py

```diff
@@ -1197,6 +1202,11 @@ def test_reverse_foreignkey_fields(self):
         results = yield s2.backlinks.foreignkeymodels()
         self.assertEqual(sorted(results), ["bar1", "bar2"])
 
+        s2 = yield simple_model.load("foo")
+        results = yield s2.backlinks.foreignkeymodel_keys()
+        self.assertEqual(sorted(results), ["bar1", "bar2"])
+        self.assertEqual(results.has_next_page(), False)
```

Comment:
The `has_next_page()` check is there to make sure we get something that looks like an index page. The actual index page implementations are tested elsewhere and don't need to be tested again here.